packages/shared/src/consts.ts

```diff
@@ -87,5 +87,3 @@ export const MIN_TICK = -887272;
 export const MAX_TICK = -MIN_TICK;
```

Comment:
would be great to replace the `CHANGELOG` entry with this:
```
- `SwapSDK.prototype.getQuoteV2` returns a `recommendedSlippageTolerancePercent` 
  property now. The value is calculated based on current market conditions to prevent 
  refunds while protecting against big price movements. It can be passed into
  the `requestDepositAddressV2` method to set the slippage tolerance for a swap.
```